src/test/java/com/google/api/codegen/protoannotations/testdata/library_v2_gapic.yaml

```diff
@@ -28,16 +28,8 @@ language_settings:
     package_name: Google.Example.Library.V1
 interfaces:
   - name: google.example.library.v1.MyProto
-    retry_codes_def:
```

Comment:
All the changes here were supposed to be in https://github.com/googleapis/gapic-generator/pull/2724